components/constants/pref_names.cc

```diff
@@ -84,15 +81,11 @@ const char kBraveShieldsSettingsVersion[] = "brave.shields_settings_version";
 const char kDefaultBrowserPromptEnabled[] =
     "brave.default_browser_prompt_enabled";
 
-#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
#if is removed to remove a mess with deps.
IMHO it's ok to not use `BUILDFLAG` for simple constant declarations.